_harp/agency/services/prototyping.md

```diff
@@ -1,86 +1,50 @@
++ A startup looking to create an [MVP](http://en.wikipedia.org/wiki/Minimum_viable_product)
++ A business looking to test out a new idea
++ An employer looking to recruit developers
++ Any project that may have a positive impact on the world
 
-The intention of the prototyping service is to help our graduates build a portfolio, showcase their extensive capabilities and build client/employer relationships. We offer this service to ambitious individuals and businesses where we feel there is a good chance of repeat work or employment—particularly once clients understand what our graduates are capable of producing.
+## Your questions answered
 
-Typically, we offer this service to:
+*What sort of prototypes do you build?*  
+We build working web applications. In other words, software applications that run in a web browser.
+
+*What about mobile phone apps, for iPhone and Android?*  
+We can certainly build native apps, but when building a prototype we strongly recommend starting with a browser-based application. Doing it this way will allow you to test your ideas on mobile without blowing your development budget.
+
+*What's the first step?*  
+Come and tell our developers about your idea.
+
+*What happens next?*  
+One of our development teams will arrange a follow-up meeting to agree on a *minimum set of deliverables* and a start date for your project.
+
+*What is the minimum set of deliverables?*  
+We have two weeks to deliver your prototype. Before we start, We will make a commitment to deliver the minimum that you require in order to be able to successfully test your product idea.
+
+*How soon will my prototype be delivered?*  
+Your team will aim to deliver your prototype to the agreed specification within approximately 10 working days of the agreed start date (if it takes longer you will not be charged more).
+
+*What support do you provide after delivery of the prototype?*  
+After a grace period during which you can report any bugs in the delivered software, you will need to retain one or more of the team on a freelance basis to get continued support.
+
+*What happens if I want further work done on my prototype?*  
+We encourage you to retain one or more of the development team to continue to work on your project on a freelance or permanent basis after delivery of the prototype.
+
+*Who owns the IP on my prototype?*  
+You will own the intellectual property. Your development team will handover all the software, documented and tested, on completion of the project.
+
+*What is the advantage of this service?*  
+The great advantage of this service is that it will give you an opportunity to work with some very talented developers early in their careers. If you are looking to hire developers or build a team for a new project, you are unlikely to find a better source of affordable talent in London.
+
+*What if I am past the MVP or prototype stage and am ready to build a scalable, fully-functioning application?*  
```

Comment:
I'm not entirely sure this is accurate as we talk about MVPs at every client meeting and I would like to expand on our relationship, but we can iterate on this :) :+1: